contracts/WildcardSteward_v3.sol

```diff
@@ -270,16 +267,13 @@ contract WildcardSteward_v3 is Initializable {
                 tokenAuctionBeginTimestamp[i]
             );
             // Adding this after the add token emit, so graph can first capture the token before processing the change artist things
-            changeArtistAddressAndCommission(
-                tokens[i],
-                _artists[i],
-                _artistCommission[i]
-            );
-
-            // // No need to initialize this.
-            // if (timeLastCollectedBenefactor[benefactor] == 0) {
-            //     timeLastCollectedBenefactor[benefactor] = now;
-            // }
+            if (_artists.length > i) {
+                changeArtistAddressAndCommission(
+                    tokens[i],
+                    _artists[i],
+                    _artistCommission[i]
+                );
+            }
```

Comment:
You've written it so that no artist commission is set for "end few" tokens? I see

Reply:
Yeah, if you want to launch half the tokens without a commission you don't waste the gas on that. Sorry, I meant to comment but forgot to submit them...